src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  let totalCost = days * dailyCost;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Use a constant for the number of days and the discount amount to avoid hardcoding values directly in the expressions. This makes the code more maintainable and the logic behind the discount clearer.